doc/adr/0007-bundesident-url-scheme.md

```diff
@@ -0,0 +1,29 @@
+# 7. Switch from eid:// to bundesident:// scheme
```

Comment:
I'd rather be more generic here, so e.g. "Use own url scheme" or "Custom ClientURL scheme". What do you think?